client/js/join-channel.js

```diff
@@ -20,8 +20,8 @@ function toggleButton(network) {
 	// Toggle content of tooltip
 	const tooltip = network.find(".add-channel-tooltip");
 	const altLabel = tooltip.data("alt-label");
-	tooltip.data("alt-label", tooltip.attr("aria-label"));
-	tooltip.attr("aria-label", altLabel);
+	tooltip.data("alt-label", tooltip.prop("aria-label"));
+	tooltip.prop("aria-label", altLabel);
```

Comment:
I've read the SO link you gave, but still unclear/not convinced: why are we changing from `attr` to `prop`? AFAIK `attr` is not deprecated.

Reply:
As far as I understood it, `prop` is used to directly access DOM properties, which we are doing here, and should generally be faster.

`prop` is basically `element[ property ] = value;` while `attr` is `element.setAttribute` and whole lot of other magic.

I changed it everywhere in the code to be consistent, as we have mixed attrs/props before.